.github/scripts/get-version.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -e
+
+grep -Eo "var semanticConventionsVersion = \".*\"" build.gradle.kts | grep -Eo "[0-9]+.[0-9]+.[0-9]+"
```

Comment:
Modified to find the version from `build.gradle.kts`, and needs to pick out the correct version since there are many matches for the `[0-9]+.[0-9]+.[0-9]+` pattern in that file.